src/lib/import-export/export/exporters/default-exporter.ts

```diff
@@ -160,20 +160,54 @@ export class DefaultExporter extends EventEmitter implements Exporter {
 			const folderName = category === 'muPlugins' ? 'mu-plugins' : category;
 			const absolutePath = path.join( this.options.site.path, 'wp-content', folderName );
 			const archivePath = path.relative( this.options.site.path, absolutePath );
-			this.archive.directory( absolutePath, archivePath, ( entry ) => {
-				const fullArchivePath = path.join( archivePath, entry.name );
-				const isExcluded = this.pathsToExclude.some( ( pathToExclude ) =>
-					fullArchivePath.startsWith( path.normalize( pathToExclude ) )
-				);
-				if (
-					isExcluded ||
-					entry.name.includes( '.git' ) ||
-					entry.name.includes( 'node_modules' )
-				) {
-					return false;
+
+			const hasSpecificSelections =
+				this.options.specificSelections &&
+				( ( category === 'plugins' && this.options.specificSelections.plugins ) ||
+					( category === 'themes' && this.options.specificSelections.themes ) );
+
+			if ( hasSpecificSelections ) {
+				const selectedItems =
+					category === 'plugins'
+						? this.options.specificSelections!.plugins!
+						: this.options.specificSelections!.themes!;
+
+				for ( const itemName of selectedItems ) {
```

Comment:
`!` is not a good practice and better to try to avoid using it, since we lie to TS that it's not an optional field.
But, actually we can remove it and duplicatrion of the condition and simplify the code at the same time with:
```suggestion
			const partialFolderItems =
				this.options.specificSelections &&
				( ( category === 'plugins' && this.options.specificSelections.plugins ) ||
					( category === 'themes' && this.options.specificSelections.themes ) );

			if ( partialFolderItems ) {
				for ( const itemName of partialFolderItems ) {
```